PRIVACY_MODEL.md

```diff
@@ -0,0 +1,61 @@
+This proposal introduces new metaprogramming concepts on private fields and methods. This document describes the invariants, guarantees and scope of this privacy as a whole, in light of this feature.
+
+## Private fields and methods are *hidden*
+
+Previous documentation used the term "hard private" to explain the guarantees, but that term may lead some to false conclusions. Here, we use the term *hidden*, defined as follows:
+
+> A class element is *hidden* with the syntax `#name`, which makes it inaccessible except to those granted explicit access to it.
+
+In this proposal, the following mechanisms grant access to hidden elements:
+- Being lexically contained inside the place where the hidden element is defined, e.g., `class { #x; method() { this.#x } }`, the body of `method()` can see `#x`.
+- Decorating an element, e.g., in `class { @dec #x; #y; }`, `@dec` can see `#x` but not `#y`.
+- Decorating a class containing the element, e.g., in `@dec class { #x; #y; }`, `@dec` can see `#x` and `#y`.
+
+Future proposals may also grant access to hidden elements, but this will always be through a mechanism which is syntactically apparent; implicit access will never be granted. For example, a `protected` contextual keyword could be added to classes which grants access to subclasses, and this access grant would be similar to that which decorators do.
```

Comment:
I think this is a poor choice of example that will distract from the point being made. Can we pick a less controversial one?

Reply:
Sure, do you have any suggestions?

Reply:
i suppose we could have some kind of `grant(otherFunction)` syntax? I also agree that decorators would be enough; maybe the example should just be omitted entirely?

Reply:
I'm not sure what you mean by `grant`.

I'd like to make a statement about language evolution. Do you think I should omit this statement entirely? Or, do you have an idea for how to rephrase it?

Reply:
I think the statement is fine; I'm concerned about the specific example.